hazelcast/src/main/java/com/hazelcast/spi/NamedOperation.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spi;
+
+public interface NamedOperation {
```

Comment:
I forgot to add Javadoc, will be added. It is for operations that carries the name of the data structure that they are going to work on. We already have `name` field most of the operations. I just pulled to an interface to use it to provide map level granularity on quorum. Because I need to know which map is going to be updated before a put operation that I can check if quorum is activated for that map.